docs/guides/StandarizedUnitsNaming.md

```diff
@@ -0,0 +1,119 @@
+# Standardowe nazewnictwo jednostek
+
+
+
+## Wstęp
+
+W Armaforces przyjęło się stosowanie zwyczajowego, standardowego nazewnictwa jednostek. Należy pamiętać, że to nie jest jedyne nazewnictwo jakie missionmaker może wprowadzić w swojej misji. 
+
+Jako przykład dla rzadziej używanych jednostek (inne niż np. Alpha, Bravo, Kilo, November, Zulu, itd.), nazewnictwa nie raz różniło się między różnymi misjami oraz także między różnymi missionmakerami (np. Mike - transport lądowy zamiast Mike - transport powietrzny). 
```

Comment:
this sound weird and looks super random writed.

Reply:
Sorry, you are right - "Gotta write fast" method was used - redacted. Thanks!